plugins/out_cloudwatch_logs/cloudwatch_logs.c

```diff
@@ -552,6 +560,13 @@ static struct flb_config_map config_map[] = {
      " be created)"
     },
 
+    {
+     FLB_CONFIG_MAP_BOOL, "auto_retry_requests", "false",
+     0, FLB_FALSE, 0,
+     "Automatically retry requests to AWS services upon request failure. (If true, requests will be"
```

Comment:
Let's re-word this because "requests will be retried at most once per each failed request." is true but also misleading. The customer will still get retries with backoff from the fluent bit engine. 

I prefer:

```
Immediately retry failed requests to AWS services once. This option does not affect the normal Fluent Bit retry mechanism with backoff. Instead, it enables an immediate retry with no delay for networking errors, which may help improve throughput when there are transient/random networking issues. 
```

Something like that. @zhonghui12 thoughts? Let's change the doc string for all the plugins.

Reply:
Okay. This sounds good. Added changes.